java/src/jmri/jmrix/loconet/LnPacketizer.java

```diff
@@ -493,17 +491,22 @@ public void startThreads() {
     @SuppressWarnings("deprecation") // stop() is deprecated, but it's not going away
     @Override
     public void dispose() {
```

Comment:
Shouldn’t the dispose method just call terminateThreads and then super.dispose?  There are quite a few duplicated lines of code here.

Reply:
Yep, good idea. Soon after the current build fails again...;-)

Reply:
Wow no error, will my luck hold?